modules/sdk-coin-bsc/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,160 @@
+import { getBuilder } from '../getBuilder';
+import { Ecdsa, TransactionType } from '@bitgo/sdk-core';
+import { KeyCombined } from 'modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types';
+
+describe('BSC Transfer Builder', () => {
+  describe('TSS Signing with 2 of 3', function () {
+    let MPC: Ecdsa;
+    let A_combine: KeyCombined, B_combine: KeyCombined, C_combine: KeyCombined;
+    let serializedTransaction: Buffer;
+
+    before('key generation and unsigned tx building', async () => {
+      MPC = new Ecdsa();
+      const A = await MPC.keyShare(1, 2, 3);
+      const B = await MPC.keyShare(2, 2, 3);
+      const C = await MPC.keyShare(3, 2, 3);
+
+      A_combine = MPC.keyCombine(A.pShare, [B.nShares[1], C.nShares[1]]);
+      B_combine = MPC.keyCombine(B.pShare, [A.nShares[2], C.nShares[2]]);
+      C_combine = MPC.keyCombine(C.pShare, [A.nShares[3], B.nShares[3]]);
+
+      const txBuilder = getBuilder('tbsc');
+      txBuilder.type(TransactionType.SingleSigSend);
```

Comment:
Is this type correct?

Reply:
it unfortunately is not i think -- i think singlesigsend is reserved for fee address recoveries/sends